students/Jaidjen/lesson03/Mailroom1.py

```diff
@@ -0,0 +1,73 @@
+import sys  # imports go at the top of the file
+
+
+Mail  =[('James Smith',[2000.25, 500, 27.50]),
+        ('Dorothy Parker',[3175, 475.45, 92.30]),
+        ('Dohgyu Hwang',[10670, 350.19,175]),
+        ('Sally Yu', [100, 925]),
+        ('Jenny Ramos',[25.14])]
+
+prompt = "\n".join(("Functions:",
+          "Please choose from below options:",
+          "1 - Send a Thank You email",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+
+
+def view_mail(Mail):
+    print("{:20s}{:20s}{:20s}{:20s}".format("DONOR ", "| TOTAL GIVEN ", "| NUMBER OF GIFTS ", "| AVERAGE GIFTS "))
+    print("-----------------------------------------------------------------------------")
+
+    for i in range(len(Mail)):
+        donate=[]
+        dnr_ave= sum(Mail[i][1])
+        ave_amt= int(len(Mail[i][1]))
+        tot_ave= dnr_ave/ave_amt
+        donate.append(dnr_ave)
+        sorted(donate)
+        print("{:20s}{:^20.2f}{:^19d}{:^18.2f}".format(Mail[i][0], dnr_ave, len(Mail[i][1]), tot_ave))
+
+def add_mail():
+    print("If you want a list of donors please type 'list' ")
+    new_mail = input("Please enter the full name of the donor? ")
+    for i in range(len(Mail)):
```

Comment:
this is not correct loop structure here - should be a while True - you are only using `i` when user selects "list"